cyhy/mailer/CyhyNotificationMessage.py

```diff
@@ -58,16 +62,20 @@ class CyhyNotificationMessage(ReportMessage):
 
 <p>Cyber Hygiene scans of your host(s) conducted in the past day have detected one or both of the following:
```

Comment:
Mirroring my earlier suggestion:
```suggestion
<p>Cyber Hygiene scans of your host(s) conducted in the past day have detected one or more of the following:
```

Reply:
All good here :)